src/jsCommonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/process.kt

```diff
@@ -0,0 +1,7 @@
+package it.krzeminski.snakeyaml.engine.kmp.internal
+
+public external val process: Process
+
+public external interface Process {
+    public val env: dynamic
+}
```

Comment:
issue: This does not work for `wasmJs` but `wasmJs` depends on `jsCommon` sources. And this causes compilation errors. We should move it to `jsMain` sources

Reply:
Thanks, I couldn't understand what jsCommonMain means, now I see it's shared between regular and WASM JS!